src/Core/src/Handlers/SwipeView/SwipeItemMenuItemHandler.Android.cs

```diff
@@ -0,0 +1,213 @@
+
+
+		public SwipeItemMenuItemHandler() : base(Mapper, CommandMapper)
+		{
+
+		}
+
+		protected SwipeItemMenuItemHandler(IPropertyMapper mapper, CommandMapper? commandMapper = null)
+			: base(mapper, commandMapper ?? CommandMapper)
+		{
+		}
+
+		public SwipeItemMenuItemHandler(IPropertyMapper? mapper = null) : base(mapper ?? Mapper)
+		{
+
+		}
+
+		protected override void ConnectHandler(AView nativeView)
+		{
+			base.ConnectHandler(nativeView);
+			NativeView.ViewAttachedToWindow += OnViewAttachedToWindow;
+		}
+
+		void OnViewAttachedToWindow(object? sender, AView.ViewAttachedToWindowEventArgs e)
+		{
+			UpdateSize();
+		}
+
+		protected override void DisconnectHandler(AView nativeView)
+		{
+			base.DisconnectHandler(nativeView);
+			nativeView.ViewAttachedToWindow -= OnViewAttachedToWindow;
+		}
+
+		public static void MapSource(SwipeItemMenuItemHandler handler, ISwipeItemMenuItem image) =>
+			MapSourceAsync(handler, image).FireAndForget(handler);
+
+		public static Task MapSourceAsync(SwipeItemMenuItemHandler handler, ISwipeItemMenuItem image)
+		{
+			return handler.SourceLoader.UpdateImageSourceAsync();
+		}
+
+		public static void MapTextColor(SwipeItemMenuItemHandler handler, ITextStyle view)
+		{
+			(handler.NativeView as TextView)?.UpdateTextColor(view);
+		}
+
+		public static void MapCharacterSpacing(SwipeItemMenuItemHandler handler, ITextStyle view)
+		{
+			(handler.NativeView as TextView)?.UpdateCharacterSpacing(view);
+		}
+
+		public static void MapFont(SwipeItemMenuItemHandler handler, ITextStyle view)
+		{
+			var fontManager = handler.GetRequiredService<IFontManager>();
+
+			(handler.NativeView as TextView)?.UpdateFont(view, fontManager);
+		}
+
+		public static void MapText(SwipeItemMenuItemHandler handler, ISwipeItemMenuItem view)
+		{
+
+			(handler.NativeView as TextView)?.UpdateTextPlainText(view);
+
+			handler.UpdateSize();
+		}
+
+		public static void MapBackground(SwipeItemMenuItemHandler handler, ISwipeItemMenuItem view)
+		{
+			handler.NativeView.UpdateBackground(handler.VirtualView.Background);
+
+			var textColor = GetSwipeItemColor(handler.VirtualView.Background?.ToColor());
+
+			if (handler.NativeView is TextView textView)
+			{
+				if (textColor != null)
+					textView.SetTextColor(textColor.ToNative());
+
+				textView.TextAlignment = ATextAlignment.Center;
+			}
+		}
+
+		public static void MapVisibility(SwipeItemMenuItemHandler handler, ISwipeItemMenuItem view)
+		{
+			var swipeView = handler.NativeView.Parent.GetParentOfType<MauiSwipeView>();
+			if (swipeView != null)
+				swipeView.UpdateIsVisibleSwipeItem(view);
+
+			handler.NativeView.Visibility = view.Visibility.ToNativeVisibility();
+		}
+
+		static Color? GetSwipeItemColor(Color? backgroundColor)
+		{
+			if (backgroundColor == null)
+				return null;
+
+			var luminosity = 0.2126f * backgroundColor.Red + 0.7152f * backgroundColor.Green + 0.0722f * backgroundColor.Blue;
```

Comment:
I know this code comes from Xamarin.Forms but could be a good moment to create an extension method with this.